apps/frontend/src/App.tsx

```diff
@@ -72,29 +52,40 @@ const App: FC<Record<string, never>> = () => {
                     <Logout />
                   </Frame>
                 </Route>
-
-                <Route exact path={staticPages({}).terms.template}>
-                  <ConfiguredLayout>
-                    <Frame title={null}>
-                      <Content pageId="terms-and-conditions" />
-                    </Frame>
-                  </ConfiguredLayout>
-                </Route>
-                <Route exact path={staticPages({}).privacyPolicy.template}>
-                  <ConfiguredLayout>
-                    <Frame title={null}>
-                      <Content pageId="privacy-policy" />
-                    </Frame>
-                  </ConfiguredLayout>
-                </Route>
-
                 <Route>
                   <CheckAuth>
-                    <ConfiguredLayout>
-                      <Frame title={null}>
-                        <GuardedApp />
-                      </Frame>
-                    </ConfiguredLayout>
+                    {({ isAuthenticated }) =>
+                      !isAuthenticated ? (
```

Comment:
As we have the `GuardedApp` component, it would be nice to have the "UnGuarded" (name is wip) component

Reply:
renamed GuardedAPP to AuthenticatedApp